jena-base/src/main/java/org/apache/jena/atlas/lib/CacheFactory.java

```diff
@@ -33,7 +33,7 @@
 
     /** Create a null cache */
     public static <Key, Value> Cache<Key, Value> createNullCache() {
-        return new Cache0<>() ;
+        return new CacheGuava<>(0) ;
     }
```

Comment:
Hm. There is one use of `Cache0`. It's indirectly, via `CacheFactory`, and it occurs in `BlockMgrCache` at [line 62](https://github.com/apache/jena/blob/master/jena-tdb/src/main/java/org/apache/jena/tdb/base/block/BlockMgrCache.java#L62). I'm not sure enough as to `BlockMgrCache`'s responsibilities to feel comfortable rewiring it to not require the existence of a `BlockMgrCache.readCache` without your go-ahead. I would feel better if I better understood the comment "Caches are related so we can't use a Getter for cache management." Does this refer to `readCache` and `BlockMgrCache.writeCache`?